modules/ocha_ai_chat/src/Controller/OchaAiChatController.php

```diff
@@ -55,6 +55,12 @@ public function __construct(Connection $connection, RequestStack $request_stack)
    */
   public function statistics(RouteMatchInterface $route_match, Request $request) {
     $response = [];
+
+    if ($this->access($this->currentUser())->isForbidden()) {
+      $response = '{"error": "Access denied!"}';
```

Comment:
```suggestion
      $response = ["error" => "Access denied!"];
```

By default JsonResponse expects unencoded data. Without that, this is double encoded: `"{\u0022error\u0022: \u0022Access denied!\u0022}"`.

The alternative, if the string is preferred, is to set the `$json` parameter: `JsonResponse($response, 403, json: TRUE);`.